### DISCUSSION TITLE
Need help replacing vc_redist with VCLibs

### DISCUSSION THREAD

**Discussion by AkazaRenn** [microsoft/WindowsAppSDK#3420] [Q&A] [answered]
Hi everyone, I'm developing a MSIX-packaged app that includes a hand-made DLL which requires `vc_redist` to run, otherwise the whole app will silently crash. I would like to replace `vc_redist` with `VCLibs` so the store can handle the installation, but it didn't work somehow. I've tried adding `VCLibs` into the dependencies of the packaging, but the app installed still crashes as if it doesn't exist, only `vc_redist` can make it work out. I'm quite confused why it would happen, would appreciate it if anyone have any suggestions.

Here is the project: https://github.com/AkazaRenn/FruitLanguageSwitcher/tree/dev/vcppdep
Inside is the DLL project: https://github.com/AkazaRenn/FruitLanguageSwitcher/tree/dev/vcppdep/Core/LanguageSwitcher

Thanks in advance!

**Comment by DarranRowe** [comment, marked as answer]
If you don't have to worry about passing C++ types across the boundary, have you considered using the [Hybrid CRT](https://github.com/microsoft/WindowsAppSDK/blob/main/docs/Coding-Guidelines/HybridCRT.md) method? This does remove the need for the C++ redistributable.

**Reply by AkazaRenn** [reply]
> If you don't have to worry about passing C++ types across the boundary, have you considered using the [Hybrid CRT](https://github.com/microsoft/WindowsAppSDK/blob/main/docs/Coding-Guidelines/HybridCRT.md) method? This does remove the need for the C++ redistributable.

Good idea, thank you!
